app/views/candidate_interface/training_with_a_disability/_form.html.erb

```diff
@@ -5,20 +5,18 @@
 </h1>
 
 <p class="govuk-body">You might benefit from extra support if you’re disabled, have a mental health condition or educational needs.</p>
-<p class="govuk-body">If you choose to tell us you need support, we’ll let your training provider know. They can then make adjustments so you can attend an interview or do the training.</p>
-<p class="govuk-body">Examples of support could be:</p>
+<p class="govuk-body">Teacher training providers may be able to make adjustments, such as:</p>
 <ul class="govuk-list govuk-list--bullet">
   <li>organising equipment like a hearing loop or an adapted keyboard</li>
   <li>putting you in touch with support staff if you have a mental health condition</li>
   <li>making sure classrooms are wheelchair accessible</li>
 </ul>
-<p class="govuk-body">If the help you need is not covered by your provider making adjustments, you might also be able to get support from <%= govuk_link_to 'Access to Work', 'https://www.gov.uk/access-to-work' %>. This could include a grant to help cover the costs of practical support in the workplace.</p>
-<h2 class="govuk-heading-m">It’s against the law to discriminate</h2>
-<p class="govuk-body">If you’re disabled, <%= govuk_link_to 'it’s against the law to discriminate against you', t('get_into_teaching.url_training_with_a_disability') %>. Training providers must not:</p>
+<p class="govuk-body">If you’re disabled it's against the law to discriminate against you. Training providers must not:</p>
 <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-6">
   <li>ask disability or health questions if they’re not relevant to your ability to become a teacher</li>
   <li>reject your application because you’re disabled</li>
 </ul>
+<p class="govuk-body"><a href="#" class="https://getintoteaching.education.gov.uk/get-support-training-to-teach-if-you-are-disabled">Get support training to teach if you're disabled</a>.</p>
```

Comment:
This link doesn't work - I want it to go here: https://getintoteaching.education.gov.uk/get-support-training-to-teach-if-you-are-disabled

What am I doing wrong? @avinhurry

Reply:
Discussed offline

Reply:
@EmmaFrith are you happy you know what you need to do to fix this?